block/config.ml

```diff
@@ -1,6 +1,26 @@
 open Mirage
 
-let main = foreign "Unikernel.Main" (console @-> block @-> job)
+type shellconfig = ShellConfig
+let shellconfig = Type ShellConfig
+
+let config_shell = impl @@ object
+    inherit base_configurable
+
+    method configure i =
+      let dir = Info.root i in
+      Functoria_app.Cmd.run "dd if=/dev/zero of=%s/disk.img count=100000" dir
+
+    method clean i =
+      let dir = Info.root i in
+      Functoria_app.Cmd.run "rm -f %s/disk.img" dir
```

Comment:
I know @samoht wanted to replace the various misc stuff by `bos` at some point. I don't know much about those APIs, it's probably a good idea, but please open a ticket and discuss it with him.